clipper_admin/clipper_admin/deployers/cloudpickle.py

```diff
@@ -1,15 +1,20 @@
 """
```

Comment:
Pulled the latest cloudpickle. This fixes issues with recursive dependencies that caused issues when serializing pytorch models.

Reply:
If we're replacing Cloudpickle, let's just switch to using the cloudpickle pip package (`pip install cloudpickle`).

Reply:
See #343

Reply:
Done